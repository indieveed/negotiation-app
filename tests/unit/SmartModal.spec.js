import { shallowMount } from '@vue/test-utils'
import Modal from '@/components/smart/modal.vue'


describe('ui/tabs.vue', () => {
  it('renders the component', () => {
    const wrapper = shallowMount(Modal)
    expect(wrapper.html()).toMatchSnapshot()
  })
  it('renders the slots', () => {
    const wrapper = shallowMount(Modal, {
      slots: {
        body: `body`,
        header: `header`,
        footer: `footer`,
      }
    })
    expect(wrapper.contains('.modal__body')).toBe(true)
    expect(wrapper.contains('.modal__footer')).toBe(true)
    expect(wrapper.contains('.modal__header')).toBe(true)
  })
})
