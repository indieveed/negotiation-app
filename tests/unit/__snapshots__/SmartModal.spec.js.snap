// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui/tabs.vue renders the component 1`] = `
<transition-stub name="modal" appear="true">
  <div class="modal"><a href="#close" aria-label="Close" class="modal__overlay"></a>
    <div class="modal__container"><button href="#close" aria-label="Close" class="btn modal__close"></button>
      <div class="modal__header"></div>
      <div tabindex="0" class="modal__body"></div>
      <!---->
    </div>
  </div>
</transition-stub>
`;
